enclave_manager/avalon_enclave_manager/avalon_enclave_info.py

```diff
@@ -98,7 +97,15 @@ def __update_sig_rl():
 
 
 # -----------------------------------------------------------------
-def initialize_with_configuration(config):
+def initialize_enclave(enclave_config):
+    """initialize_enclave -- Call the initialization function on the
+    enclave module
+    """
+    return _initialize_with_configuration(enclave_config)
```

Comment:
Can we move the function `_initialize_with_configuration()` body here and delete `_initialize_with_configuration()`